bims/templates/map_page/map-templates.html

```diff
@@ -444,6 +444,8 @@ <h5 class="modal-title">Upload a boundary</h5>
     <li class="ui-state-default" style="display: <%= display %>"
         data-name="<%= id %>">
         <div class="layer-selector-tags">
+            <span value="<%= key %>" class="badge badge-primary layer-source"><%= source %></span>
```

Comment:
we can just remove `<%= source %>` if you insert the value by accessing the classname

Reply:
This source is the writing within the blue - do you want to remove it?